util/stmtsummary/statement_summary_test.go

```diff
@@ -755,20 +746,21 @@ func newStmtSummaryReaderForTest(ssMap *stmtSummaryByDigestMap) *stmtSummaryRead
 }
 
 // Test stmtSummaryByDigest.ToDatum.
-func (s *testStmtSummarySuite) TestToDatum(c *C) {
-	s.ssMap.Clear()
+func TestToDatum(t *testing.T) {
+	t.Parallel()
+	ssMap := newStmtSummaryByDigestMap()
 	now := time.Now().Unix()
 	// to disable expiration
-	s.ssMap.beginTimeForCurInterval = now + 60
+	ssMap.beginTimeForCurInterval = now + 60
 
 	stmtExecInfo1 := generateAnyExecInfo()
-	s.ssMap.AddStatement(stmtExecInfo1)
-	reader := newStmtSummaryReaderForTest(s.ssMap)
+	ssMap.AddStatement(stmtExecInfo1)
+	reader := newStmtSummaryReaderForTest(ssMap)
 	datums := reader.GetStmtSummaryCurrentRows()
-	c.Assert(len(datums), Equals, 1)
-	n := types.NewTime(types.FromGoTime(time.Unix(s.ssMap.beginTimeForCurInterval, 0)), mysql.TypeTimestamp, types.DefaultFsp)
-	e := types.NewTime(types.FromGoTime(time.Unix(s.ssMap.beginTimeForCurInterval+1800, 0)), mysql.TypeTimestamp, types.DefaultFsp)
-	t := types.NewTime(types.FromGoTime(stmtExecInfo1.StartTime), mysql.TypeTimestamp, types.DefaultFsp)
+	require.Equal(t, 1, len(datums))
+	n := types.NewTime(types.FromGoTime(time.Unix(ssMap.beginTimeForCurInterval, 0)), mysql.TypeTimestamp, types.DefaultFsp)
+	e := types.NewTime(types.FromGoTime(time.Unix(ssMap.beginTimeForCurInterval+1800, 0)), mysql.TypeTimestamp, types.DefaultFsp)
+	time := types.NewTime(types.FromGoTime(stmtExecInfo1.StartTime), mysql.TypeTimestamp, types.DefaultFsp)
```

Comment:
Using the standard package name as a variable name is not a good practice.

Reply:
indeed, fixed, thanks